universal-application-tool-0.0.1/app/views/admin/programs/ProgramApplicationListView.java

```diff
@@ -30,17 +29,20 @@ public ProgramApplicationListView(AdminLayout layout) {
   }
 
   public Content render(long programId, ImmutableList<Application> applications) {
+    // TODO: [i18n].
```

Comment:
(non-blocking): I'm not sure we need to internationalize admin-only strings, if only because we don't actually have a way for admins to choose a different language at this point. @carolinedanzi can you confirm?

Reply:
Good question! We don't have a way to do this yet, though it will be similar to the applicant side. I think eventually we can internationalize the admin side but for now the assumption is they use the default of English